bigbluebutton-html5/public/locales/en.json

```diff
@@ -336,6 +336,7 @@
     "app.presentationUploder.conversion.204": "No content to capture",
     "app.presentationUploder.upload.413": "File is too large, exceeded the maximum of {0} MB",
     "app.presentationUploder.genericError": "Oops, Something went wrong ...",
+    "app.presentationUploder.upload.fileVirus": "Upload failed: there is a problem with the file",
```

Comment:
```suggestion
    "app.presentationUploder.upload.fileVirus": "Upload failed: Virus detected! Please check your file and retry.",
```

Reply:
> app.presentationUploder.upload.fileVirus

Well... we are already calling it a virus...

I believe seeing "Upload failed: there is a problem with the file" will automatically lead to customer tickets asking Why.  The point here is to add support for virus scanning (something BigBlueButton benefits from) so I don't really see a problem showing it in action.

We could water it down a bit, perhaps instead of "Virus detected" we could say something more like "Our virus scan flagged this file as suspicious" or similar.